### ISSUE TITLE
Is it possible to add the downgrade function  ?

### ISSUE THREAD

**Issue by Poop81** [eliboa/NxNandManager#87]
Hello, having a problem with the 14.x.x update on my console, I can no longer access my emunand sd card, I end up with a boot crash, would it be possible to be able to downgrade and update from nxnandmanager? Maybe in a future update?

**Comment by Masamune3210** [comment]
Its better to figure out the issue since a lot of issues would persist even if you try to downgrade it

**Comment by Poop81** [comment]
Alright, that's pretty long.


I bought a switch V1 a year ago on the net with the nand hs. No backup was provided to me because the person did not have to make one.

I bought a complete emmc on Aliexpress and replaced it. I followed a tutorial with a nand donor to make it work, it starts in OFW and CFW.

I wanted to update my EmuMmc from 13.0.0 to 14.1.1 and unfortunately I had a boot crash afterwards.

Stupid as I am, I reversed my sd cards, I had made a backup of my sd card with NxNandManager, but I took the wrong sd card to update.

I made various attempts by creating an Emummc from 13.0.0 and I did update by update and when I get to 14.0.0, I get a boot crash.

I make the decision to update the switch to 14.1.1 via CFW SysNand and I restart it afterwards normally so that the update is official and the efuses burn and I can boot again in CFW EmuNand the sd card which boot crash .

Unfortunately, that didn't change anything.

If I forgot to provide elements tell me I will add the information.

 Thank you

**Comment by Masamune3210** [comment]
Honestly, if it's getting stuck before atmosphere takes over (ukn pkg1, stuff like that) then it's probably easier to make an issue on Hekate's issue tracker. If it's getting into atmosphere but crashing during boot, creating an issue there instead would be your best bet.

**Comment by Poop81** [comment]
I put a photo.
You advise me to open a ticket sooner. I will open it with atmoshpere I think because it is he who crashes? Thank you

**Comment by Masamune3210** [comment]
Yeah, that screen is atmosphere's crash handler, so making the issue in their repo should help. Make sure to follow the template and explain as much as you can

**Comment by Poop81** [comment]
Alright, thank you :)
